release/gff/gff.sgw.gurage/source/sgw-unilex-NewOrthography-WordList.tsv

```diff
@@ -0,0 +1,8574 @@
+﻿ለምቻ	19612
+ለፍልፎት	19612
+ሉስዮስ	19612
+ሊጥ	29418
+ላማ	19612
+ላባና	39224
+ሌዊ	58837
+ሌዊም	19612
+ሌጌዎን	19612
+ል	68643
+ልካ	19612
+ልፍ	147093
+ሎጥ	19612
+መሌከታ	29418
+መሌኩ	29418
+መሌካ	382442
+መሌካመታ	19612
+መሌካም	107868
+መሌካሮ	19612
+መሌካታ	68643
+መሌክ	735467
+መሌክም	98062
+መሌክምታ	49031

```

Comment:
I noticed some words have `` U+E044 which shows as Private-Use-Area.
Are they valid? (I don't know if the GFF keyboards can generate that character).

Reply:
@dyacob I see your latest update. Is U+E044 a valid character?

Reply:
yes, these characters are valid.  the encoding uses an experimental orthography for Gurage that is being tested in the user community at present.